.do/deploy.template.yaml

```diff
@@ -0,0 +1,9 @@
+spec:
+  name: docker-stellar-core-horizon
+  services:
+  - dockerfile_path: Dockerfile
+    git:
+      branch: acharb-quickdeploy
+      repo_clone_url: https://github.com/stellar/docker-stellar-core-horizon.git
+    name: docker-stellar-core-horizon
+    run_command: /start --standalone
```

Comment:
I think you need to use the `--randomize...` option here?

Reply:
yup thx for catching :)
https://github.com/stellar/docker-stellar-core-horizon/pull/276/commits/945031e66fd0d1223167c92e4ade9bccdfa48561